managers/views.py

```diff
@@ -1,8 +1,9 @@
+from django.db.models.functions import Lower
```

Comment:
This function just make string which is composed with Upper letter to lower letters?

Reply:
Yes, but it does it on database side, during SQL query.